### PR TITLE
DOCS-6466 X-Forwarded-Path leading slash

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-6466 X-Forwarded-Path leading slash

### CODE REVIEW

**Review by joaquinstan — APPROVED** [review]
🎸 🚀 🎸

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 1]
```diff
@@ -65,11 +65,11 @@ For example:
 
 Add headers for each route described above.
```
Comment (kbostrom) [inline modules/ROOT/pages/portal-vanity-domain.adoc]: GLOBALLY avoid using "above" or "below."
See Style Ref.

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 2]
```diff
@@ -65,11 +65,11 @@ For example:
...
 
-[source,console,linenums]
-----
-'X-Forwarded-Path': Base path of your vanity domain public portal URL. This can be any path. It can be just '/' for the root path. It does not need to match the base path for the portal in the anypoint.mulesoft.com domain. This path must end with a slash ("/"). Example: 'path/of/justice/'
-'X-Forwarded-Host': Your public portal vanity domain. Example: 'developers.robocop.com'
-----
+* `X-Forwarded-Path` is the base path of your vanity domain public portal URL, such as `/path/of/justice/`.
```
Comment (kbostrom) [inline modules/ROOT/pages/portal-vanity-domain.adoc:68]: Assuming that the audience knows what a "vanity domain" is.

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 3]
```diff
@@ -65,11 +65,11 @@ For example:
...
++
+It can be any path that begins and ends with a slash ("/"), and it can be just `/` for the root path. It does not need to match the base path for the portal in the anypoint.mulesoft.com domain.
```
Comment (kbostrom) [inline modules/ROOT/pages/portal-vanity-domain.adoc]:
```suggestion
The vanity domain public portal base path can be either any path that begins and ends with a slash (`/`) or a root path represented by a single slash (`/`). It does not need to match the base path for the anypoint.mulesoft.com domain portal.
```
Please carefully reread to be sure that my suggested rewording did not change technical meaning.

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 4]
```diff
@@ -65,11 +65,11 @@ For example:
...
++
+* `X-Forwarded-Host` is your public portal vanity domain, such as `developers.robocop.com`.
```
Comment (kbostrom) [inline modules/ROOT/pages/portal-vanity-domain.adoc:72]: Note that "public portal vanity domain" in this item is reworded from "vanity domain public portal" in the item above it. Is that OK?

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc]: Fixed. Thank you!

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc:68]: Yes, this audience will know that term.

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc]: I rewrote this for better clarity and consistency and incorporated your suggestions. Thank you!

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc:72]: Yes, that is correct here. X-Forwarded-Path is the path of the portal that goes with the domain, and X-Forwarded-Host is the domain that goes with the portal.

**Pull request merged**